src/main/java/org/skriptlang/skript/bukkit/recipes/elements/ExprRecipeKey.java

```diff
@@ -0,0 +1,63 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.Keyed;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.Recipe;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Recipe Name")
+@Description("Get the namespacekey of a recipe")
```

Comment:
Small suggestion since the class is NamespacedKey 
```suggestion
@Description("Get the namespaced key of a recipe.")
```